pkg/scheduler/scheduler.go

```diff
@@ -334,6 +334,10 @@ func (s *Scheduler) Bind(args extenderv1.ExtenderBindingArgs) (*extenderv1.Exten
 	err = nodelock.LockNode(args.Node)
 	if err != nil {
 		klog.ErrorS(err, "Failed to lock node", "node", args.Node)
+		res = &extenderv1.ExtenderBindingResult{
+			Error: err.Error(),
+		}
+		return res, nil
```

Comment:
if we return an ExtenderBindingResult in here, What is the specific performance? Where will these error messages appear and will the scheduler ignore this node in the next loop?

Reply:
I've added more details in the PR description.


> will the scheduler ignore this node in the next loop?

no.